src/player.py

```diff
@@ -3,23 +3,34 @@
 class Player(Game):
     # initialize hand with games deck
     def __init__(self, game):
+        print("I compile")
 
     # lay down, should just call players hand easily
+    # put down a full set, or add to a set that is already laid down
     def lay_down(self):
+        print("I compile")
 
     # decides whether to go out
     def go_out(self):
+        print("I compile")
 
-    # obvious
+    # put down one card to discard pile
     def discard(self):
+        print("I compile")
 
     # pickup from deck
     def pick_up(self):
+        print("I compile")
 
     # pickup from discard deck
     def pick_up_discards(self):
+        print("I compile")
 
-   
-
+    # prints hand
+    def __str__(self):
+        print("I compile")
 
+    # shows sets all players have laid down
+    def show_laydown(self):
+        print("I compile")
 
```

Comment:
what does shows sets mean?

Reply:
shows everything other players have put down

Reply:
what does it return

Reply:
i thought it would print everything each player has laid down by player? like every 'set' is enclosed by [] and is listed by player?
ex: Alex: [Q Diamonds, K Diamonds, Ace Diamonds], [6 Clubs]
Luke: [3 Clubs, 4 Clubs, 5 Clubs]

Reply:
yeah i don't see this as neccessary. Also, whenever you are making a method to print something in the future usually using the class's special method is the way to go

Reply:
see the deck idea below